website/config.toml

```diff
@@ -66,7 +66,7 @@ anchor = "smart"
 ## Site Params
 [params]
 
-latest_release_version = "0.5.1"
+latest_release_version = "0.5.2"
```

Comment:
I think this is used everywhere for all versions of docs, unfortunately. It would be nice if we somehow had access to `params.versions[something].version`. I couldn't figure this out though.

Reply:
We don't need to block on this, though.